lib/unit_tests_utils/bosh.rb

```diff
@@ -158,5 +124,14 @@ def self.parse_json_from_create_release(raw_json)
     }
   end
 
+  def self.execute_or_raise_error(command, msg)
```

Comment:
Please implement tests for the function.
See: `spec/lib/unit_tests_utils/bosh_spec.rb`

Reply:
Done